### PR TITLE
Feature/oembed bgr 4431

### PR DESCRIPTION
**Pull request by jasonKufner**
Implements rich oEmbed assertion detail view with support for embedded version v1 and v2

### CODE REVIEW

File: src/app/public/components/badge-assertion/badge-assertion.component.html [hunk 1]
```diff
@@ -10,10 +10,85 @@
...
+				<h1>{{badgeClass.name}}</h1>
+				<div class="oembedbadge-x-issuer">
+					<img [src]="assertion.image"
+					     width=48
+					     alt=""
+					     srcset=""
+					/>
+					<dl>
+						<dt>{{issuer.name}}</dt>
+						<dd>{{badgeClass.description}}</dd>
```
Comment (nqthqn) [inline src/app/public/components/badge-assertion/badge-assertion.component.html]: This should be
```
<dt>Issued By</dt>
<dd>{{issuer.name}}</dd>
```

File: src/app/public/components/badge-assertion/badge-assertion.component.html [hunk 2]
```diff
@@ -10,10 +10,85 @@
...
+				   target="_blank"
+				   rel="noopener"
+				>
+					<img [src]="assertion.image" width="200"/>
+				</a>
+			</div>
+			<div class="oembedbadge-x-summary">
+				<h1>{{badgeClass.name}}</h1>
+				<div class="oembedbadge-x-issuer">
+					<img [src]="assertion.image"
```
Comment (nqthqn) [inline src/app/public/components/badge-assertion/badge-assertion.component.html]: This should be the Issuer logo.

File: src/app/public/components/badge-assertion/badge-assertion.component.html [hunk 3]
```diff
@@ -10,10 +10,85 @@
...
+						<dd>{{badgeClass.description}}</dd>
+					</dl>
+				</div>
+				<dl class="oembedbadge-x-info">
+					<dt>Issued On</dt>
+					<dd>
+						<time [date]="assertion.issuedOn" format="mediumDate"></time>
+					</dd>
+				</dl>
+				<dl class="oembedbadge-x-info">
```
Comment (nqthqn) [inline src/app/public/components/badge-assertion/badge-assertion.component.html]: This `dl` should only be displayed if the badge assertion has an expiration date

File: src/app/public/components/badge-assertion/badge-assertion.component.html [hunk 4]
```diff
@@ -10,10 +10,85 @@
...
+		>
+			<button class="oembedbadge-x-toggle"
+			        (click)="toggleShowV2EmbedDescription()"
+			>
+				{{showV2EmbedDescription ? 'Hide Description' : 'Show Description'}}
+			</button>
+			<div class="oembedbadge-x-slideinner">
+				<div>
+					<h2>Description</h2>
+					<p class="u-text">Open Badges are verifiable, portable digital badges with embedded metadata about skills and achievements. They comply with the Open Badges Specification and are shareable across the web.
```
Comment (nqthqn) [inline src/app/public/components/badge-assertion/badge-assertion.component.html]: Should not be static. Use `{{badgeClass.description}}` (i think)

File: src/app/public/components/badge-assertion/badge-assertion.component.ts [hunk 5]
```diff
@@ -1,23 +1,41 @@
...
 @Component({
-	templateUrl: './badge-assertion.component.html'
+	templateUrl: './badge-assertion.component.html',
+	animations: [
+		trigger('openClose', [
+			state('open',
+				style({ top: '8px'})
+			),
+			state('closed',
+				style({ top: 'calc(100% - 40px)'})
```
Comment (nqthqn) [inline src/app/public/components/badge-assertion/badge-assertion.component.ts]: I did not realize we could just do this! you are brilliant. 🤦‍♂ myself. let's do this in CSS! 

- upgrade to next patch release of the PL
- only toggle -is-active class

File: src/app/public/components/badge-assertion/badge-assertion.component.html (hunk 4 above)
Comment (nqthqn) [reply src/app/public/components/badge-assertion/badge-assertion.component.html]: Also — should be truncated to 186 characters with `...` added if truncated.

File: src/app/public/components/badge-assertion/badge-assertion.component.html (hunk 1 above)
Comment (jasonKufner) [reply src/app/public/components/badge-assertion/badge-assertion.component.html]: Updated

File: src/app/public/components/badge-assertion/badge-assertion.component.ts (hunk 5 above)
Comment (nqthqn) [reply src/app/public/components/badge-assertion/badge-assertion.component.ts]: Bump `@concentricsky/badgr-style` to `17.0.6`

File: src/app/public/components/badge-assertion/badge-assertion.component.html (hunk 2 above)
Comment (jasonKufner) [reply src/app/public/components/badge-assertion/badge-assertion.component.html]: Updated. Per conversation issuer.image

File: src/app/public/components/badge-assertion/badge-assertion.component.html (hunk 3 above)
Comment (jasonKufner) [reply src/app/public/components/badge-assertion/badge-assertion.component.html]: updated

File: src/app/public/components/badge-assertion/badge-assertion.component.html (hunk 4 above)
Comment (jasonKufner) [reply src/app/public/components/badge-assertion/badge-assertion.component.html]: done

File: src/app/public/components/badge-assertion/badge-assertion.component.ts (hunk 5 above)
Comment (jasonKufner) [reply src/app/public/components/badge-assertion/badge-assertion.component.ts]: done

**Pull request merged**